databricks_template_schema.json

```diff
@@ -98,7 +112,14 @@
       "type": "string",
       "description": "\nWhether to include MLflow Recipes",
       "default": "no",
-      "enum": ["no", "yes"]
+      "enum": ["no", "yes"],
+      "skip_prompt_if": {
+        "properties": {
```

Comment:
Note specifying multiple properties is an "AND" operation, not an "OR" operation. So we can't specify that Recipes should be skipped if either Unity Catalog or Feature Store is "yes". @shreyas-goenka can we add that support?

Reply:
Filed a ticket in our internal backlog to add support for OR